frontend/src/components/collaborator/CollaboratorForm.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div>
+    <e-text-field
+      v-if="status"
+      class="ec-status-field"
+      :value="translatedStatus"
+      readonly
+      :name="$tc('entity.campCollaboration.fields.status')"
+    >
+      <template #append>
+        <slot name="statusChange" />
+      </template>
+    </e-text-field>
+    <v-tooltip v-if="readonlyRole" eager bottom>
+      <span id="readonly">
+        {{ $tc('components.collaborator.collaboratorForm.roleHint') }}
+      </span>
+      <template #activator="{ on }">
+        <div tabindex="0" class="mt-3" v-on="on">
+          <e-select
+            v-model="localCollaboration.role"
+            fieldname="role"
+            readonly
+            aria-readonly="true"
+            aria-describedby="readonly"
+            :name="$tc('entity.campCollaboration.fields.role')"
+            :items="[
+              {
+                key: 'manager',
+                role: $tc('entity.camp.collaborators.manager'),
+                abilities: $tc('global.collaborationAbilities.manager'),
+                icons: ['mdi-eye-outline', 'mdi-pencil-outline', 'mdi-cog-outline'],
+              },
+              {
+                key: 'member',
+                role: $tc('entity.camp.collaborators.member'),
+                abilities: $tc('global.collaborationAbilities.member'),
+                icons: ['mdi-eye-outline', 'mdi-pencil-outline'],
+              },
+              {
+                key: 'guest',
+                role: $tc('entity.camp.collaborators.guest'),
+                abilities: $tc('global.collaborationAbilities.guest'),
+                icons: ['mdi-eye-outline'],
+              },
+            ]"
+            :hint="$tc('components.collaborator.collaboratorForm.roleHint')"
+            item-value="key"
+            item-text="role"
+            vee-rules="required"
+          >
+            <template #selection="{ item }">
+              <span
+                >{{ item.role }} &middot;
+                <span class="grey--text"
+                  ><template v-for="icon in item.icons"
+                    ><v-icon :key="icon" x-small>{{ icon }}</v-icon
+                    >&thinsp;</template
+                  ></span
+                >
+              </span>
+            </template>
+          </e-select>
+        </div>
+      </template>
+    </v-tooltip>
+    <e-select
+      v-else
+      v-model="localCollaboration.role"
+      :name="$tc('entity.campCollaboration.fields.role')"
+      fieldname="role"
+      :items="[
+        {
+          key: 'manager',
+          role: $tc('entity.camp.collaborators.manager'),
+          abilities: $tc('entity.camp.collaborators.managerAbilities'),
```

Comment:
This key is not translated

<img width="1013" alt="image" src="https://github.com/ecamp/ecamp3/assets/449555/d00410d3-48f8-4951-bdf7-5d09e0e1395b">